modules/ROOT/examples/swift/code-snippets/SampleCodeTest.swift

```diff
@@ -57,6 +57,20 @@ class SampleCodeTest {
         Database.setLogLevel(.verbose, domain: .query)
         // end::logging[]
     }
+    
+    func dontTestFileLogging() throws {
+        // tag::file-logging[]
+        let tempFolder = NSTemporaryDirectory().appending("cbllog")
+        Database.log.file.config = LogFileConfiguration.init(directory: tempFolder)
```

Comment:
not much difference, but more easy = `LogFileConfiguration(directory: tempFolder)`

Reply:
Yes. Updated in https://github.com/couchbase/docs-couchbase-lite/commit/ae30b84c9253d44e5ed4f11101eec45813c5b6e9